EasyPost/ClientManager.cs

```diff
@@ -35,7 +35,7 @@ internal static Client Build()
         {
             if (getCurrent == null)
             {
-                throw new ClientNotConfigured();
+                throw new ClientNotConfigured("No function set to retrieve the current client.");
```

Comment:
What's this?

Reply:
Oops, that was just a cleanup thing that accidentally slipped in here. I guess ClientNotConfigured was never throwing an error message before, just the exception.

Reply:
Did we still need this? Sounds like you were going to remove it?